bank-sdk/example-app/src/androidTest/java/net/gini/android/bank/sdk/exampleapp/ui/screens/DigitalInvoiceScreen.kt

```diff
@@ -134,55 +145,88 @@ class DigitalInvoiceScreen {
         }
     }
 
-    fun verifyHelpTextOnNextScreen() {
+    fun verifyHelpTextOnNextScreen(): Boolean {
         val helpText = device.findObject(
             UiSelector()
                 .className("android.widget.TextView")
                 .text("Help")
         )
-        helpText.exists()
+        return helpText.exists()
     }
 
-    fun verifyFirstTitleOnHelpScreen() {
+    fun verifyFirstTitleOnHelpScreen(): Boolean {
+        var isFirstTitleDisplayed = false
         onView(
             allOf(withId(net.gini.android.bank.sdk.R.id.gbs_help_title),
                 withText("1. How does a digital invoice work?")
             )
         )
-            .check(matches(isDisplayed()))
+            .check { view, noViewFoundException ->
+                if (noViewFoundException == null || view.isShown()) {
```

Comment:
[Clarification] Same comment as further up, shouldn't we return true only if there is no exception AND the view is shown ?